specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -456,107 +453,107 @@ The Java™ Authentication and Authorization
 Service (JAAS) enables services to authenticate and enforce access
 controls upon users. It implements a Java technology version of the
 standard Pluggable Authentication Module (PAM) framework and supports
-user-based authorization. The Java™ Authorization Service Provider
-Contract for Containers (JACC) defines a contract between a Java EE
+user-based authorization. Jakarta™ Authorization
+defines a contract between a Jakarta EE
 application server and an authorization service provider, allowing
-custom authorization service providers to be plugged into any Java EE
-product. The Java™ Authentication Service Provider Interface for
-Containers (JASPIC) defines an SPI by which authentication providers
+custom authorization service providers to be plugged into any Jakarta EE
+product. Jakarta™ Authentication
+defines an SPI by which authentication providers
 implementing message authentication mechanisms may be integrated in
-client or server message processing containers or runtimes. The Java EE
-Security API leverages JASPIC, but provides an easier to use SPI for
+client or server message processing containers or runtimes. Jakarta
+Security leverages Jakarta Authentication, but provides an easier to use SPI for
 authentication of users of web applications and defines identity store
 APIs for authentication and authorization.
 
 === Web Services
 
-Java EE provides full support for both clients
+Jakarta EE provides full support for both clients
 of web services as well as web service endpoints. Several Java
-technologies work together to provide support for web services. The Java
-API for XML Web Services (JAX-WS) and the Java API for XML-based RPC
-(JAX-RPC) both provide support for web service calls using the SOAP/HTTP
+technologies work together to provide support for web services. Jakarta
+XML Web Services and Jakarta XML-based RPC
```

Comment:
Jakarta XML Web Services is not part of Jakarta EE 8.  This needs to stay "JAX-WS".
Ditto several places below.

The JAX-RPC change is correct.